crates/ruff_linter/src/rules/flake8_bandit/rules/suspicious_function_call.rs

```diff
@@ -877,7 +911,11 @@ pub(crate) fn suspicious_function_call(checker: &mut Checker, call: &ExprCall) {
         }
     }
 
-    let Some(diagnostic_kind) = checker.semantic().resolve_qualified_name(call.func.as_ref()).and_then(|qualified_name| {
+    if checker.semantic().in_annotation() {
+        return;
+    }
```

Comment:
Just to confirm, this is because now we check all references which can come from type annotations as well.